packages/lambda-xyz/src/wmts-capability.ts

```diff
@@ -0,0 +1,162 @@
+    version: '1.0.0',
+};
+
+const LayerPreamble: Record<string, string> = {
+    aerial: `
+<ows:Title>NZ Aerial Imagery Basemap</ows:Title>
+<ows:Abstract></ows:Abstract>
+<ows:BoundingBox crs="urn:ogc:def:crs:EPSG::2193">
+  <ows:LowerCorner>3327016.96 -1000000.0</ows:LowerCorner>
+  <ows:UpperCorner>10000000.0 3086983.68</ows:UpperCorner>
+</ows:BoundingBox>
+<ows:WGS84BoundingBox>
+  <ows:LowerCorner>-180.0 -85.0511287798</ows:LowerCorner>
+  <ows:UpperCorner>180.0 85.0511287798</ows:UpperCorner>
+</ows:WGS84BoundingBox>
+<ows:Identifier>nz_aerialimagery_basemap</ows:Identifier>
+<Style>
+  <ows:Identifier>default</ows:Identifier>
+</Style>
+<Format>image/png</Format>
+<TileMatrixSetLink>
+  <TileMatrixSet>global-mercator</TileMatrixSet>
+</TileMatrixSetLink>
+`,
+};
+
+const ProviderInfo = `
+  <ows:ServiceIdentification>
+    <ows:Title>National Base Mapping Service (LINZ)</ows:Title>
+    <ows:Abstract>
+        National Mapping Service provided by Land Information New Zealand
+    </ows:Abstract>
+    <ows:ServiceType>OGC WMTS</ows:ServiceType>
+    <ows:ServiceTypeVersion>1.0.0</ows:ServiceTypeVersion>
+    <ows:Fees>
+        There are no fees associated with data access via the web interface, API
+        or Web Services.
+    </ows:Fees>
+    <ows:AccessConstraints>
+        LINZ Data Service Customers complete a self registration process where
+        they accept site terms. Access to the data is subject to the terms of
+        the Creative Commons 3.0
+    </ows:AccessConstraints>
+  </ows:ServiceIdentification>
+  <ows:ServiceProvider>
+    <ows:ProviderName>Land Information New Zealand</ows:ProviderName>
+    <ows:ProviderSite xlink:href="http://www.linz.govt.nz"/>
+    <ows:ServiceContact>
+      <ows:IndividualName>LINZ Customer Support</ows:IndividualName>
+      <ows:PositionName>Customer Support</ows:PositionName>
+      <ows:ContactInfo>
+        <ows:Phone>
+          <ows:Voice>+64 4 4600110</ows:Voice>
+          <ows:Facsimile>+64 4 4983842</ows:Facsimile>
+        </ows:Phone>
+        <ows:Address>
+          <ows:DeliveryPoint>Land Information New Zealand</ows:DeliveryPoint>
+          <ows:City>Wellington</ows:City>
+          <ows:PostalCode>6145</ows:PostalCode>
+          <ows:Country>New Zealand</ows:Country>
+          <ows:ElectronicMailAddress>customersupport@linz.govt.nz</ows:ElectronicMailAddress>
+        </ows:Address>
+      </ows:ContactInfo>
+    </ows:ServiceContact>
+  </ows:ServiceProvider>
+`;
+
+const CommonGlobalMercator = `<TopLeftCorner>-20037508.3428 20037508.3428</TopLeftCorner>
+<TileWidth>256</TileWidth>
+<TileHeight>256</TileHeight>`;
+
+const MatrixSets: Record<string, Record<string, () => string>> = {
+    aerial: {
+        'global-mercator'(): string {
+            let ans = '';
+            let scale = 559082264.029;
+            for (let i = 0; i < 20; ++i, scale *= 0.5) {
+                const dim = String(i + 1);
+                ans +=
```

Comment:
We should leave everything as a Vdom node until the end otherwise the formatting will be weird